src/neptune_scale/api/run.py

```diff
@@ -695,6 +695,44 @@ def log_files(
         *,
         timestamp: Optional[datetime] = None,
     ) -> None:
+        """Appends a file value and uploads the file contents at a particular step.
```

Comment:
> Appends a file value

Should we maybe explain that we're appending to a file series attribute?

Reply:
We can perhaps illustrate this better in the docs.